src/ui_fsmenu/about.cc

```diff
@@ -30,16 +30,16 @@ constexpr int16_t kPadding = 4;
 
 FullscreenMenuAbout::FullscreenMenuAbout(FullscreenMenuMain& fsmm)
    : UI::Window(&fsmm,
+                UI::WindowStyle::kFsMenu,
                 "about",
-                (fsmm.get_w() - calc_desired_window_width(fsmm)) / 2,
-                (fsmm.get_h() - calc_desired_window_height(fsmm)) / 2,
-                calc_desired_window_width(fsmm),
-                calc_desired_window_height(fsmm),
+                (fsmm.get_w() - fsmm.calc_desired_window_width("about")) / 2,
```

Comment:
Use en enum class as flags for this, or a boolean. What shall we call it? I think `bool is_option_window` should to it.